### ISSUE TITLE
Style output cells differently

### ISSUE THREAD

**Issue by AtmaMani** [greenape/mknotebooks#3]
This is a great project which I plan to use heavily. One little defect is bothering me. The appearance of input and output cells are identical making reading hard. (See pic of the included example below) 
![image](https://user-images.githubusercontent.com/6750179/57256163-c47cae00-700a-11e9-91c2-12bb5d69d4ed.png)

Is there a way (such as using the preamble processor) to contrast their appearance? A subtle highlight or removal of the background gray box would help.

**Comment by greenape** [comment]
You can definitely use the preamble-processor to achieve this, e.g:

```python
get_ipython().display_formatter.formatters["text/markdown"].for_type(
    str, lambda x: f">`{x}`"
)
```

Would also be possible to instate some defaults along those lines in the plugin, but basically the code is just calling Jupyter's notebook exporter behind the scenes.

**Comment by AtmaMani** [comment]
I tried, but does not seem to make any difference. My preamble script looks like below:
```python
import pandas as pd
import tabulate as tabulate

def to_md(self):
    return tabulate.tabulate(self.head(), self.columns, tablefmt="pipe")

get_ipython().display_formatter.formatters["text/html"].for_type(pd.DataFrame, to_md)
get_ipython().display_formatter.formatters["text/markdown"].for_type(
    str, lambda x: f">`{x}`")
```
I appended your suggestion to the end of the `pandas_to_markdown.py` file. My understanding is this file will be imported which will cause it to execute it fully. I tried commenting out and manipulating the string formatting, but see no difference. I may need a bit more pointers. Thanks.

**Comment by becauseofAI** [comment]
rp @greenape
Have you solved the problem? @AtmaMani

**Comment by AtmaMani** [comment]
@becauseofAI no, I have not. I concluded that to get notebook output cells to appear differently from input cells, it is best to export it to HTML directly, instead of via markdown. In this process, I settled with Nikola static website builder as it has builtin support for notebooks.

**Comment by greenape** [comment]
Closed via #10
